projects/browser-extension/vue-app/src/__old_views/google.view.vue

```diff
@@ -154,7 +154,7 @@ export default {
       return metaData[type];
     },
     getImageUrl(img) {
```

Comment:
This is a function with the same names as on line 180. I assume one of them can be removed?

Reply:
Dit komt door het mergen. Ik ga deze file gewoon verwijderen, dit was volgens mij bedoelt als back-up maar deze file heeft nooit een new/oud variant gehad.